components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -89,6 +89,15 @@ internal class DisplayToolbar(
         setOnClickListener(null)
     }
 
+    internal val titleView = AppCompatTextView(context).apply {
+        id = R.id.mozac_browser_toolbar_title_view
+        gravity = Gravity.CENTER_VERTICAL
+        textSize = URL_TEXT_SIZE
+        visibility = View.GONE
+
+        setSingleLine(true)
```

Comment:
Do we want to define here what happens if the text is longer than the view's width?
-> https://developer.android.com/reference/android/widget/TextView.html#setEllipsize(android.text.TextUtils.TruncateAt)

Reply:
Updated this, though I wonder why the `url` doesn't also have this set?

Reply:
The url sets a fading edge in the `init {}` block.